nix/overlays/nixpkgs-overrides.nix

```diff
@@ -3,4 +3,5 @@ self: super: {
   # We can *nearly* replace this with upstream nixpkgs, but unfortunately we also need a patch
   # that hasn't been merged upstream yet. And you can't override the pieces of a 'bundlerApp'.
   asciidoctor= super.callPackage ../asciidoctor { };
+  z3 = super.callPackage ../z3.nix {};
```

Comment:
Does this mean that we could e.g. replace the line in `lambda.nix` with `z3 = pkgs.z3.override { staticbin = true }`? Given the new z3 will be in `pkgs`, and therefore presumably in `pkgsMusl`?

Reply:
we can I think